app/subscriber/src/components/tool-bar/utils/toInstanceContent.ts

```diff
@@ -0,0 +1,25 @@
+import { IContentModel, IReportInstanceContentModel, IReportModel } from 'tno-core';
+
+/**
+ * Converts an array of content to an array of report instance content.
+ * @param content The content to convert.
+ * @param report The report to convert the content for.
+ * @param sectionName The section name to convert the content for.
+ */
+export const toInstanceContent = (
```

Comment:
I would simplify.  You should already know the instance at this point.

Use these arguments `instanceId, sectionName, content, sortOrder`